### PR TITLE
Upgrade jmxfetch to latest version

### PR DESCRIPTION
**Pull request by mar-kolya**
https://github.com/DataDog/jmxfetch/releases/tag/0.34.0

### CODE REVIEW

File: dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle [hunk 1]
```diff
@@ -4,10 +4,7 @@ plugins {
 apply from: "${rootDir}/gradle/java.gradle"
 
 dependencies {
-  compile('com.datadoghq:jmxfetch:0.32.1') {
-    exclude group: 'org.slf4j', module: 'slf4j-log4j12'
-    exclude group: 'log4j', module: 'log4j'
```
Comment (tylerbenson) [inline dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle]: We don't want to import their usage of log4j2, so we need a similar set of excludes here.

Comment (mar-kolya) [reply dd-java-agent/agent-jmxfetch/agent-jmxfetch.gradle]: should be fixed now

**Comment by mar-kolya** [comment]
> Please verify their logging is being sent to our slf4j implementation instead of their logger.
this seems to be working

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-all.zip
```
Comment (tylerbenson) [inline gradle/wrapper/gradle-wrapper.properties]: This is probably better in a separate PR, but I'll let it slide.

Comment (mar-kolya) [reply gradle/wrapper/gradle-wrapper.properties]: thanks :)

**Pull request merged**